plugin.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 
 // ConfigHeaders defines the headers to use for the different values.
 type ConfigHeaders struct {
+	Version string `json:"version,omitempty"`
 	Cipher string `json:"cipher,omitempty"`
```

Comment:
```suggestion
	Version string `json:"version,omitempty"`
    Cipher  string `json:"cipher,omitempty"`
```

this is what the linter is mad about